### ISSUE TITLE
Content files can't contain """

### ISSUE THREAD

**Issue by antoine-atmire** [dillonkearns/elm-pages#26]
If any content file contains triple double quotes `"""` elm-pages will fail to compile.

Example content file:

```
---
title: txt format
type: page
---

Just some plain text
"""
some more plain text
```

```
I was partway through parsing a record, but I got stuck here:

11812|     , { frontMatter = """{"title":"txt format","type":"page"}
11813| """ , body = Just """
11814| Just some plain text
11815| """
          ^
I was expecting to see a closing curly brace next. Try putting a } next and see
if that helps?
```

Some context for my use case: I'm making a site to show my personal git repositories, a bit like github/gitlab does, but in a static/prerendered way. This will include the code and the diffs as well. As you can imagine some Elm files have `"""` in them.

**Comment by dillonkearns** [comment]
Interesting use case! Yeah, this should definitely be escaped. Thanks for filing the issue!

**Comment by dillonkearns** [comment]
Fixed by #84. Thanks again for reporting the issue!
